draft-ietf-dnsop-svcb-https.md

```diff
@@ -498,6 +502,12 @@ is the effective TargetName:
     svc2.example.net. 300   IN A     192.0.2.2
     svc2.example.net. 300   IN AAAA  2001:db8::2
 
+Some authoritative DNS implementations restrict the owner names of A and AAAA
+records.  For example, BIND 9.18 requires these owner names to be hostnames
+when the `check-names` option is enabled {{CHECK-NAMES}}.  The ServiceMode
+TargetName is required to point at A or AAAA records, so any such restrictions
+may impact the choice of TargetName, and may preclude use of the value "." if
+the ServiceMode record's owner name is not permitted for A and AAAA records.
```

Comment:
Nit: Quotation marks, "some versions BIND".

I think "some versions of BIND" should be "BIND in some configurations", since this is an optional feature.

"may not" no longer seems appropriate when we have explicitly identified a configuration where this applies.  "do not" seems better.

"due to various interpretations of RFCs" seems too informal to appear in an RFC.  Perhaps we can remove it.

"could be an operational issue" seems both too informal and extremely vague.  I'm not even sure it's true.  The operator MUST publish the A/AAAA records before publishing the TargetName, so this issue will be addressed when choosing the owner name for those records, and the TargetName will be wherever they were ultimately placed.

Reply:
No.

This document was already approved and sent to the RFC Editor; I clawed it back because there were concerns expressed that there was a significant error, and, after WG discussions we agreed to the patch in #399.

Because it was purely operational advice there seemed to be general agreement that Erik's initial text was OK to add (and I got IESG agreement that we could include it).
I understand the desire to massage and improve this text, fix the nits and loose threads, etc - but it's too late for that.

So, please incorporate only the original, agreed to text and post a new revision, so that I can send it to the IESG for (re)approval, and return it to the RFC Editor queue. 
Sorry to be inflexible (and grumpy), but this is all post WGLC, IETF LC, IESG eval and IESG approval - now is not the time to be making additional changes.

Please see: https://mailarchive.ietf.org/arch/msg/dnsop/LhEufm0a8IjoXZ_ck0b9RTSkDTc/